### PR TITLE
Test for repo.includepkgs behavior when installing local RPM

### PR DESCRIPTION
**Pull request by jan-kolarik**
Requires: https://github.com/rpm-software-management/dnf5/pull/1435.

### CODE REVIEW

File: dnf-behave-tests/dnf/excludes-includes.feature [hunk 1]
```diff
@@ -133,3 +133,14 @@ Scenario: Install RPMs that are in includepkgs in main conf and NOT in excludepk
     And Transaction is following
         | Action        | Package                          |
         | install       | flac-libs-0:1.3.2-8.fc29.x86_64  |
+
+
+@bz2275530
+Scenario: Install local RPM when repository includepkgs are restricted
+  Given I use repository "dnf-ci-fedora"
+    And I successfully execute dnf with args "download http-parser"
+   When I execute dnf with args "install http-parser-2.4.0-1.fc29.x86_64.rpm --setopt=dnf-ci-fedora.includepkgs=pkg*"
```
Comment (m-blaha) [inline dnf-behave-tests/dnf/excludes-includes.feature:142]: You don't need to download the package, you can use something like
```
When I execute dnf with args "install {context.dnf.fixturesdir}/repos/dnf-ci-fedora/x86_64/http-parser-2.4.0-1.fc29.x86_64.rpm --setopt=dnf-ci-fedora.includepkgs=pkg*"
```

Comment (j-mracek) [reply dnf-behave-tests/dnf/excludes-includes.feature:142]: Good point

Comment (jan-kolarik) [reply dnf-behave-tests/dnf/excludes-includes.feature:142]: Sure, I'll rework it.

**Comment by j-mracek** [comment]
LGTM

**Pull request merged**